packages/backend/src/models/OpenIdIdentitiesModel.ts

```diff
@@ -84,7 +86,14 @@ export class OpenIdIdentityModel {
                 issuer: id.issuer,
                 email: id.email,
                 createdAt: id.createdAt,
-            }));
+            }))
+            .reduce(
+                (acc, curr) => ({
+                    ...acc,
+                    [curr.issuerType]: [...(acc[curr.issuerType] || []), curr],
+                }),
+                { okta: [], google: [], oneLogin: [] },
```

Comment:
I was also thinking of creating a issuerType enum to be used as a value for these. What do we think? 🤔